src/DIRAC/WorkloadManagementSystem/Service/JobManagerHandler.py

```diff
@@ -151,16 +155,18 @@ def export_submitJob(self, jobDesc):
             jobDesc = "%s]" % jobDesc
 
         # Check if the job is a parametric one
-        jobClassAd = ClassAd(jobDesc)
-        result = getParameterVectorLength(jobClassAd)
+        jobClassAd = transformParametricJobIntoParsableOne(ClassAd(jobDesc))
```

Comment:
It's because we need to upload the XML jobDescription file on the client side, so all the InputSandboxes are uploaded together as a tarball.

Reply:
> It's because we need to upload the XML jobDescription file on the client side, so all the InputSandboxes are uploaded together as a tarball.

This does not fully answer the above question, right?

Reply:
We discussed it together, but I can explain more if you desire